include/Ark/Compiler/MacroExecutors/MacroExecutorPipeline.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef ARK_COMPILER_EXECUTORPIPELINE_HPP
```

Comment:
Don't forget to add doxygen like documentation comment on top of each file, in this format (also small coding style, put a blank line before the header guards and the includes):
```cpp
/**
 * @file Lexer.hpp
 * @author Alexandre Plateau (lexplt.dev@gmail.com)
 * @brief Tokenize ArkScript code
 * @version 0.1
 * @date 2020-10-27
 * 
 * @copyright Copyright (c) 2020
 * 
 */

#ifndef ARK_COMPILER_LEXER_HPP
#define ARK_COMPILER_LEXER_HPP

#include <blabla>

code...
```